getClients.js

```diff
@@ -0,0 +1,22 @@
+exports.getAll = function(event, context, callback) {
+  const connection = db.connect(event.stageVariables);
+  var query = 'SELECT * FROM Clients ORDER BY' + event.queryStringParameters.sort';
+
+  connection.query(query, function (error, results, fields) {
+    if (error) {
+      db.close(connection);
```

Comment:
```db.close(connection)``` is duplicate in error and success; Can be moved as 1st line of the query's callback